app/views/rooms/room_tickets/new.html.erb

```diff
@@ -5,5 +5,5 @@
 <br>
 
 <div>
```

Comment:
How about replacing h1 header with this
```
<h1>Submit a TDX Ticket</h1>
<h2><%= "#{@building.name}, room #{@room.room_number}" %></h2>
```
Add `@building = @room.floor.building` into the new method in the room_tickets_controller